Extensions/Terraform/Src/Tasks/TerraformTaskV1/task.json

```diff
@@ -0,0 +1,318 @@
+{
+    "id": "FE504ACC-6115-40CB-89FF-191386B5E7BF",
+    "name": "TerraformCustomTask",
+    "friendlyName": "Custom Terraform Task",
+    "description": "Execute terraform cli commands",
+    "author": "Akash Chakrabarti",
+    "helpMarkDown": "",
+    "category": "Deploy",
+    "visibility": [
+        "Release"
+    ],
+    "demands": [
+        "terraform"
+    ],
+    "version": {
+        "Major": "0",
+        "Minor": "0",
+        "Patch": "85"
+    },
+    "instanceNameFormat": "$(provider) - terraform $(command)",
+    "execution": {
+        "Node": {
+            "target": "src/index.js"
+        }
+    },
+    "groups": [
+        {
+            "name": "backendAzureRm",
+            "displayName": "AzureRM Backend Configuration",
+            "isExpanded": false,
+            "visibleRule": "provider = azurerm && command = init && backendTypeAzureRM = azurerm"
+        },
+        {
+            "name": "backendAWS",
+            "displayName": "AWS Backend Configuration",
+            "isExpanded": false,
+            "visibleRule": "provider = aws && command = init && backendTypeAWS = aws"
+        },
+        {
+            "name": "backendGCP",
+            "displayName": "GCP Backend Configuration",
+            "isExpanded": false,
+            "visibleRule": "provider = gcp && command = init && backendTypeGCP = gcp"
+        }
+    ],
+    "inputs": [{
+            "name": "provider",
+            "type": "picklist",
+            "label": "Provider",
+            "defaultValue": "azurerm",
+            "required": true,
+            "helpMarkDown": "The name of the provider used in the terraform configuration files.<br>Only one provider must be specified.",
+            "options": {
+                "azurerm": "azurerm",
+                "aws": "aws",
+                "gcp": "gcp"
+            },
+            "properties": {
+                "EditableOptions": "False"
+            }
+        },
+        {
+            "name": "command",
+            "type": "pickList",
+            "label": "Command",
+            "defaultValue": "init",
+            "required": true,
+            "helpMarkDown": "The terraform command to execute.",
+            "options": {
+                "init": "init",
```

Comment:
@RoopeshNair  - Feedback that came in from @madhuv-msft . Should we consider a command that implicitly does init & apply together? User doesn't have to add the task twice.